crates/sui-json-rpc/src/read_api.rs

```diff
@@ -117,14 +117,38 @@ impl ReadApiServer for ReadApi {
     async fn get_objects_owned_by_address(
         &self,
         address: SuiAddress,
-    ) -> RpcResult<Vec<SuiObjectInfo>> {
-        Ok(self
+        // exclusive cursor if `Some`, otherwise start from the beginning
+        options: Option<SuiObjectDataOptions>,
+        cursor: Option<ObjectID>,
+        limit: Option<usize>,
+    ) -> RpcResult<ObjectsPage> {
+        let limit = cap_page_objects_limit(limit);
+        let options = options.unwrap_or_default();
+
+        let mut objects = self
             .state
-            .get_owner_objects(address)
-            .map_err(|e| anyhow!("{e}"))?
-            .into_iter()
-            .map(SuiObjectInfo::from)
-            .collect())
+            .get_owner_objects(address, cursor, limit + 1)
+            .map_err(|e| anyhow!("{e}"))?;
+
```

Comment:
Can you add a TODO comment here to `multi-get-object` for content/storage-rebate if it opts.show_content is true?